client/blocks/inline-help/contextual-help.js

```diff
@@ -905,6 +905,24 @@ const contextLinksForSection = {
 			),
 		},
 	],
+	hosting: [
+		{
+			link: localizeUrl( 'https://en.support.wordpress.com/sftp/' ),
```

Comment:
I couldn't find the actual slugs for these two links - took post_ids from one of Ian's posts, but can't access the actual pages - I probably don't have editor access to the help pages, so if someone with access can verify the two post_ids and slugs and let me know, or update yourself, please :-)

Reply:
I have access, so I updated the slugs with actual permalinks (post ids were correct). I also changed the titles so they mache the actual support page title (seems to be the pattern with other pages).